ehttp/Cargo.toml

```diff
@@ -25,11 +25,23 @@ all-features = true
 ## Support streaming fetch
 streaming = ["dep:wasm-streams", "dep:futures-util"]
 
+multipart = ["dep:rand", "dep:mime", "dep:mime_guess", "dep:getrandom"]
+
+json = ["dep:serde", "dep:serde_json"]
```

Comment:
Please add docstrings to these using `##`. You can see them in the docs if you run `cargo doc -p ehttp --all-features --open`.

Reply:
Oh, and sort the features